ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1345,9 +1349,16 @@ def execute_expiring_check(check):
         cert = x509.load_certificate_list(certfile)
         cert_expiry = cert[0].not_valid_after
 
-        for service in ('chronyd', 'pki_tomcatd',):
+        # Stop chronyd so it doesn't freak out with time so off
+        for service in ('chronyd',):
```

Comment:
Nitpick, can be simplified by removing the `for` + directly calling `restart_service(self.master, 'chronyd')`

Reply:
The current times should be ok I think. I extended them because I was adding sleeps and retrys to the certmonger code while trying to find the root cause.

To double-check I'll drop the time extension with this push but still run it five times and we'll see if they all still pass and don't time out.